cmd/admin/migrate.go

```diff
@@ -101,7 +101,7 @@ var MigrateCmd = &cli.Command{
 				if err != nil {
 					return errors.WithStack(err)
 				}
-				fmt.Printf("Current migration: " + last + "\n")
+                fmt.Printf("Current migration: %s\n", last)
```

Comment:
test runner was complaining about this, I'm actually not 100% sure why but it seems happy now

Reply:
ok this seems to be a legit vet error :+1: